lighthouse-extension/app/src/popup.js

```diff
@@ -15,20 +15,38 @@
  * limitations under the License.
  */
 
+const configJSON = require('../../../lighthouse-core/config/default.json');
```

Comment:
I don't think this'd work in a built extension. Let's copy the default.json into /app in a gulp script.